amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseRenewer.java

```diff
@@ -358,6 +359,10 @@ public boolean updateLease(
                 success = true;
                 return updatedLease;
             }
+        } catch (Exception e) {
```

Comment:
I am a little bit concerned that we catch this on every exception. Is it not possible that there are exceptions that we would not want to revert the lease on? Shouldn't we only catch this if the DDB update fails?

Reply:
Also even if this is the case now wouldn't it be better to define them in case another case comes along where the exception woudn't stop the lease from updating.

Reply:
I agree with your concern, updated to be only exceptions thrown when failing to update DDB